pegasus/routes/dev_routes.rb

```diff
@@ -0,0 +1,27 @@
+BUILD_STARTED_PATH = deploy_dir('build-started')
+
+# Used to restart builds on staging/test via Slack /start-build slash command.
+post '/api/dev/start-build' do
+  # Forbidden in production because it's a dev route
+  # Forbidden in development because it won't work anyway
+  forbidden! if [:development, :production].include? rack_env
+
+  dont_cache
+
+  forbidden! unless params[:token] == CDO.slack_start_build_token
```

Comment:
Non-200 responses result in a user-facing error ([docs](https://api.slack.com/slash-commands#responding_to_a_command)).  That's fine - we want to be clear that this doesn't work in certain environments, and not without a valid token.